lib/stores/GCSDataStore.ts

```diff
@@ -141,11 +121,11 @@ class GCSDataStore extends DataStore {
           return
         }
 
-        let bytes_received = (data as any).size
-        readable.on('data', (buffer: any) => {
+        let bytes_received = data.size as number
```

Comment:
Why do we need `as number` here? Shouldn´t we be able to control the return type from getOffset to always contain this numeric property?

Reply:
`size` technically can be undefined, but it's a bit weird with TS to say that in a certain function it can't be, while in other it can. Hence the cast in places it must be defined.